addons/website_sale_stock_wishlist/models/__init__.py

```diff
@@ -1,5 +1,4 @@
 # -*- coding: utf-8 -*-
 # Part of Odoo. See LICENSE file for full copyright and licensing details.
 
-from . import product_wishlist
```

Comment:
You probably need to rever this change.

Reply:
Even if product_wishlist.py is not in website_sale_stock_wishlist anymore 
<img width="522" alt="Screenshot 2022-04-05 at 14 17 09" src="https://user-images.githubusercontent.com/101560055/161751803-3f4e51b2-03b9-4af9-b7c7-6945da1c0b2a.png">
?